features/tournament/optional_place_3_match.feature

```diff
@@ -0,0 +1,32 @@
+Feature: Tournament optional place 3 match
+  As a user
+  In order to easily create a tournament
+  I want to make the place 3 match optional
+
+  Scenario: Create match without place 3 match
+    Given a user
+    And the user is logged in
+    When he visits the create event path
+    And he clicks 'New tournament'
```

Comment:
You could compress this to "he creates a tournament"

Reply:
I'm against that. That would hide the fact that here a button gets clicked

Reply:
How about "he visits the create tournament form"

Reply:
I don't think, that it would add any value. Can we just leave it as is?